tests/test_webhook/test_aiohtt_server.py

```diff
@@ -181,10 +181,20 @@ def handle_message(msg: Message):
             "aiogram.dispatcher.dispatcher.Dispatcher.silent_call_request",
             new_callable=AsyncMock,
         ) as mocked_silent_call_request:
+            method_called_event = asyncio.Event()
+            mocked_silent_call_request.side_effect = (
+                lambda *args, **kwargs: method_called_event.set()
+            )
+
             handler_event.clear()
             resp = await self.make_reqest(client=client)
             assert resp.status == 200
             await asyncio.wait_for(handler_event.wait(), timeout=1)
+            await asyncio.wait_for(method_called_event.wait(), timeout=1)
```

Comment:
FYI, a simple `asyncio.sleep(0.00000000...1)` would be enough as well. It just needs to be non-zero so that another task is scheduled before our assert. However, using a side effect and waiting for an event is more robust to future changes or inconsistencies across different installations.